src/core/core.scale.js

```diff
@@ -857,11 +857,15 @@ module.exports = function(Chart) {
 
 					var label = itemToDraw.label;
 					if (helpers.isArray(label)) {
-						for (var i = 0, y = 0; i < label.length; ++i) {
+						var lineCount = label.length / 2;
+						var lineHeight = tickFont.size * 1.5;
+						var y = me.isHorizontal() ? 0 : -lineHeight * lineCount / 1.2;
```

Comment:
I think the correct calculation depends on the actual text baseline:

a. if `textBaseline: 'top'` then `y = - lineHeight * lineCount / 2;`
b. if `textBaseline: 'middle'` then `y = - lineHeight * (lineCount - 1) / 2;`
c. if `textBaseline: 'bottom'` then `y = - lineHeight * (lineCount - 2) / 2;`

By default, `textBaseline === 'middle'` (line 738), so `b.` should work in our case?

Reply:
Oh ok makes sense, `b` does work in this case. Is it necessary to adjust the code for each `textBaseline` value?

Reply:
Since it's always `middle` for vertical scale, I would only implement `b.` for now.